backend/server/src/main/java/com/issuetracker/domain/IssuesNumber.java

```diff
@@ -0,0 +1,14 @@
+package com.issuetracker.domain;
+
+public class IssuesNumber {
+
+    private final Long[] issueNumber;
```

Comment:
```suggestion
public class IssuesNumbers {

    private final Long[] issueNumbers;
```

리스트라는 것을 명시하기 위해, 복수형으로 작성하시면 좋을 것 같습니다.

Reply:
f598710e887afabf70ffa92cd75a055487a864a4 에서 수정완료했습니다!